src/Resources/public/js/startup.js

```diff
@@ -205,6 +210,12 @@ pimcore.plugin.customermanagementframework = Class.create(pimcore.plugin.admin,
         }
 
         this.addSegmentAssignmentTab(object, 'object', type);
+        document.dispatchEvent(new CustomEvent(pimcore.events.postOpenObject, {
```

Comment:
we should not re-dispatch events here. Instead add event listeners in initialize like this https://github.com/pimcore/advanced-object-search/pull/169/files#diff-109b95b1376bdd4064598c08caf2593a3472983b6b7c4c4951f8428b66fdc6cfR23

Reply:
Should this be done for all the startup methods?